x-pack/plugins/security_solution/server/search_strategy/timeline/factory/events/all/helpers.ts

```diff
@@ -55,9 +54,9 @@ const mergeTimelineFieldsWithHit = <T>(
                 ...get('node.data', flattenedFields),
                 {
                   field: fieldName,
-                  value: specialFields.includes(esField)
-                    ? toStringArray(get(esField, hit))
-                    : toStringArray(get(esField, hit._source)),
+                  value: specialFields.includes(fieldName)
+                    ? toStringArray(get(fieldName, hit))
```

Comment:
```suggestion
                    ? toStringArray(hit[fieldName])
```

Reply:
You will get a type error because of the definition of hit `hit: { fields: Record<string, unknown[]> }` but we do know that we need some special fields like `_id`, `_index` etc...